packages/tests/e2e/robot/tests/api/API_toolchain.robot

```diff
@@ -76,3 +76,35 @@ Delete mlflow-gpu toolchain
 
 Check that toolchains do not exist
     Command response list should not contain id  toolchain  ${MLFLOW}  ${MLFLOW_GPU}
+
+#############################
+#    NEGATIVE TEST CASES    #
+#############################
+Try Create Toolchain that already exists
+    [Tags]                      negative
+    [Setup]                     Cleanup resource  toolchain-integration  ${MLFLOW}
+    [Teardown]                  Cleanup resource  toolchain-integration  ${MLFLOW}
+    Call API                    toolchain post  ${RES_DIR}/valid/mlflow_update.json
+    ${EntityAlreadyExists}      Format EntityAlreadyExists  ${MLFLOW}
+    Call API and get Error      ${EntityAlreadyExists}  toolchain post  ${RES_DIR}/valid/mlflow_update.json
+
+Try Update not existing and deleted Toolchain
+    [Tags]                      negative
+    ${WrongHttpStatusCode}      Format WrongHttpStatusCode  ${MLFLOW_NOT_EXIST}
```

Comment:
I would suggest to rename `Format WrongHttpStatusCode` to something like `Format EntityNotFound`. So it would be in the same style as `Format EntityAlreadyExists`. 
Moreover, I would refactor it as follows, but it is up to you:
- Replace `Format WrongHttpStatusCode` **keyword** by `404NotFoundTemplate` **variable**
- Use `format string ${404NotFoundTemplate} ${MLFLOW_NOT_EXIST}` in test cases
From my point of view it is more transparent and simple logic

Reply:
done